change-notes/1.23/analysis-javascript.md

```diff
@@ -17,6 +17,7 @@
 | Unused index variable (`js/unused-index-variable`)                        | correctness                                                       | Highlights loops that iterate over an array, but do not use the index variable to access array elements, indicating a possible typo or logic error. Results are shown on LGTM by default. |
 | Loop bound injection (`js/loop-bound-injection`)                          | security, external/cwe/cwe-834                                      | Highlights loops where a user-controlled object with an arbitrary .length value can trick the server to loop indefinitely. Results are not shown on LGTM by default. |
 | Suspicious method name (`js/suspicious-method-name-declaration`)          | correctness, typescript, methods                                  | Highlights suspiciously named methods where the developer likely meant to write a constructor or function. Results are shown on LGTM by default. |
+| Shell command with uncontrolled environment values (`js/shell-command-injection-from-environment`) | correctness, security, external/cwe/cwe-078, external/cwe/cwe-088 | Highlights shell commands that may change behavior inadvertently depending on the execution environment, indicating a possible violation of [CWE-78](https://cwe.mitre.org/data/definitions/78.html). Results are shown on LGTM by default.|
```

Comment:
I'm not sure about the "uncontrolled environment values" bit in the query name. I assume you mean that the values taken from the environment aren't further sanitised or checked, but it might be misunderstood to imply that you mean that the values themselves are uncontrolled. Perhaps something more neutral along the lines of "Command built from environment values" would be preferable? I'm also not 100% happy with the "environment values" terminology, though I do understand that it's not just environment variables we are talking about.